### DISCUSSION TITLE
Whats the difference between WingetUI.Installer.exe and UnigetUI.Installer.exe ?

### DISCUSSION THREAD

**Discussion by mooms06** [Devolutions/UniGetUI#3112] [Q&A]
I can see that even for the [freshly released version 3.1.5](https://github.com/marticliment/UniGetUI/releases), there are still two installers, why, and what's the difference between theme ?
It's unnecessary confusing IMHO.

**Comment by marticliment** [comment]
Old updaters still pull the installer from `WingetUI.Installer.exe`, so I can't remove them (yet)
